### PR TITLE
Fix phone login

### PR DESCRIPTION
**Pull request by dorshay6**
## ℹ️ Change description:
Nothing here

<details><summary><sub>templates<sub></summary>
<p>
  
<sub>[templates: [bug](?expand=1&template=bug.md), [feature](?expand=1&template=feature.md)]</sub>

</p>
</details>

### CODE REVIEW

File: examples/client/Locomotion/src/Components/PhoneNumberInput/index.js [hunk 1]
```diff
@@ -4,65 +4,47 @@ import { AsYouType } from 'libphonenumber-js';
...
-  useEffect(() => {
-    if (value) {
-      onChangeText(value.substring(prevCountryCode.length));
-    }
-  }, [countryCode]);
-
   const setIsoCode = async () => {
     const mobileIso = await getInputIsoCode();
-    changeCountryCode(mobileIso);
+    setDefaultCode(process.env.OVERWRITE_COUNTRY_CODE || mobileIso);
```
Comment (avivbarniv) [inline examples/client/Locomotion/src/Components/PhoneNumberInput/index.js]: Use react native config

File: examples/client/Locomotion/src/Components/PhoneNumberInput/index.js [hunk 2]
```diff
@@ -4,65 +4,47 @@ import { AsYouType } from 'libphonenumber-js';
...
       value={value}
       autoFocus={autoFocus}
       defaultCode={defaultCode}
-      onChangeText={onChangeText}
+      onChangeFormattedText={onChangeText}
       textInputProps={{
         onFocus: () => setIsFocused(true),
         onBlur: () => setIsFocused(false),
       }}
-      onChangeCountry={onChangeCountry}
```
Comment (lieblichaaron) [inline examples/client/Locomotion/src/Components/PhoneNumberInput/index.js:65]: we still need this functionality if the they dont want to use the default code no?

**Review by lieblichaaron — CHANGES_REQUESTED** [review]

File: examples/client/Locomotion/src/Components/PhoneNumberInput/index.js (hunk 1 above)
Comment (dorshay6) [reply examples/client/Locomotion/src/Components/PhoneNumberInput/index.js]: yes fixed

File: examples/client/Locomotion/src/Components/PhoneNumberInput/index.js (hunk 2 above)
Comment (dorshay6) [reply examples/client/Locomotion/src/Components/PhoneNumberInput/index.js:65]: onChangeFormattedText has both country code and actual number

**Review by lieblichaaron — APPROVED** [review]

File: examples/client/Locomotion/src/Components/PhoneNumberInput/index.js (hunk 2 above)
Comment (lieblichaaron) [reply examples/client/Locomotion/src/Components/PhoneNumberInput/index.js:65]: ah cool

**Pull request merged**